suites/pacific/cephadm/tier0_5_1_cephadm.yaml

```diff
@@ -0,0 +1,728 @@
+tests:
```

Comment:
IMO, let's have the file name as `tier_0_cephadm_5.1.yaml`

Reply:
As per our discussion, separated out build-version values against test suite name.